.github/workflows/smartbear-issue-comment-created.yml

```diff
@@ -50,13 +50,19 @@ jobs:
           
     - name: Add Comment 
       if: steps.search.outputs.issue == '' && steps.create.outputs.issue != ''
-      uses: actions/github-script@v5
+      uses: actions/github-script@v6
       with:
         github-token: ${{secrets.GITHUB_TOKEN}}
         script: |
           github.rest.issues.createComment({
             issue_number: context.issue.number,
             owner: context.repo.owner,
             repo: context.repo.repo,
-            body: `👋 Thanks, this ticket has been added to the PactFlow team's backlog as ${{steps.create.outputs.issue}}`
+            body: "👋 Hi! 'smartbear-supported' label has just been added to this issue, which will create an internal tracking ticket in PactFlow's Jira (${{steps.create.outputs.issue}}). We will use this to prioritise and assign a team member to this task. All activity will be public on this ticket. For now, sit tight and we'll update this ticket once we have more information on the next steps. <br/><br/>See <a href='https://docs.pact.io/help/smartbear'>documentaiton</a> for more information."
```

Comment:
```suggestion
            body: "👋 Hi! A 'smartbear-supported' label has just been added to this issue, which will create an internal tracking ticket in PactFlow's Jira (${{steps.create.outputs.issue}}). We will use this to prioritise and assign a team member to this task. All activity will be public on this ticket. For now, sit tight and we'll update this ticket once we have more information on the next steps. <br/><br/>See our <a href='https://docs.pact.io/help/smartbear'>documentation</a> for more information."
```

Reply:
Looks good! There was a minor spelling mistake, and have also added a few minor adjustments.